Makefile.devel

```diff
@@ -309,51 +278,49 @@ distcompile: FORCE
 
 ############################################################
 ##
-## tag:    tag the CVS sources of working directory with CVS_RELEASENAME,
-##	   set version stamp in proof-site.el and ProofGeneral.spec 
+## tag:    tag the sources of working directory with GIT_RELEASENAME,
+##	   set version stamp in proof-site.el and ProofGeneral.spec
 ##         to VERSION, and edit $(DOWNLOADHTMLS)
 ##	   if VERSION matches PRERELEASE_TAG.
 ##
-tag:		
+tag:
 	@echo "*************************************************"
-	@echo " Tagging sources... (you must rerun if CVS source dirty)"
+	@echo " Tagging sources... "
 	@echo "*************************************************"
-# Update the sources, this is almost always what we want to do.
-	if [ -z "$(NOCVS)" ] && [ -n "`cvs -n -q update -Pd | grep '^[MC] '`" ]; then cvs update -Pd; exit 1; fi
+	if [ -z "$(NOCVS)" ]; then git fetch origin && git log --exit-code master...origin/master || exit 1; fi
 # Update version in proof-site.el
 	(cd generic; mv $(VERSIONFILE) $(VERSIONFILE).old; sed -e 's/defconst $(VERSIONVARIABLE) \".*\"/defconst $(VERSIONVARIABLE) \"Proof General Version $(FULLVERSION).\"/g' $(VERSIONFILE).old > $(VERSIONFILE); rm $(VERSIONFILE).old)
 # Tag ProofGeneral.spec
 	(cd etc; mv ProofGeneral.spec ProofGeneral.spec.old; sed -e 's/Version:.*$$/Version:	$(VERSION)/g' ProofGeneral.spec.old > ProofGeneral.spec; rm ProofGeneral.spec.old)
 # Edit $(DOWNLOADHTMLS) only for prereleases.
 # Careful: the sed command below relies on previous value of PRERELEASE_TAG.
-	if [ $(PRERELEASE_TAG) = $(VERSION) ]; then \
-	  (cd $(HTMLDIR); \
-	   for f in $(DOWNLOADHTMLS); do \
-		mv $$f $$f.old; \
-		sed -e 's|ProofGeneral\([emacselc-]*\)-$(PRERELEASE_PREFIX)......|ProofGeneral\1-$(PRERELEASE_TAG)|g' $$f.old > $$f; \
-		rm $$f.old; \
-	   done) \
+#	if [ $(PRERELEASE_TAG) = $(VERSION) ]; then \
+#	  (cd $(HTMLDIR); \
+#	   for f in $(DOWNLOADHTMLS); do \
+#		mv $$f $$f.old; \
+#		sed -e 's|ProofGeneral\([emacselc-]*\)-$(PRERELEASE_PREFIX)......|ProofGeneral\1-$(PRERELEASE_TAG)|g' $$f.old > $$f; \
+#		rm $$f.old; \
+#	   done) \
 	fi
```

Comment:
This `fi` should be commented as well (the backslash in the previous line appears not to be sufficient).